enrolment-ui/src/modules/ProjectUsers/UserRequests.js

```diff
@@ -52,9 +52,9 @@ const UserRequests = props => {
                         {user.status === 'Pending' ? (
                           <div className="admin-actions">
                             <a
-                              onClick={() =>
-                                activateProjectUser(user.project, user.id, fetchAllProjectUsers)}
-                            >
+                              href="#"
```

Comment:
Links like these should've been buttons instead, but scope is already creeped as it is.
Just adding href so bootstrap styles them accordingly.